app/services/next_customer_file_reference.service.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+/**
+ * @module NextCustomerFileReferenceService
+ */
+
+const BaseNextFileReferenceService = require('./base_next_file_reference.service')
+
+class NextCustomerFileReferenceService extends BaseNextFileReferenceService {
+  /**
+   * Returns the next customer file reference for the given region and regime, in the format `nalrc50001`. See
+   * `BaseNextFileReferenceService` for further details.
+   */
```

Comment:
It's a really minor quibble but I'd move this to be above the `class` declaration as it's describing the class more than the method `_field()`.